grc/core/schema_checker/domain.py

```diff
@@ -1,9 +1,25 @@
 from .utils import Spec, expand
 
+PARAM_SCHEME = expand(
+    base_key=str,
+
+    id=str,
+    label=str,
+
+    dtype=str,
+    default=object,
+
+    options=list,
+    option_labels=list,
+    option_attributes=Spec(types=dict, required=False,
+                           item_scheme=(str, list)),
+)
+
```

Comment:
IDEA: reuse PARAM_SCHEME from block.py. Looks like, the only difference is `hide`, which would be treated as `required=False`.